fhir/fhir-connector/fhir-connector-1.0.0/org.wso2.carbon.connector/src/main/assembly/assemble-connector.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly>
+    <formats>
+        <format>zip</format>
+    </formats>
+
```

Comment:
Fixed in https://github.com/wso2-attic/esb-connectors/pull/558/commits/f9e41fea7ca349558494fc0cf418b2fbc71d3511